### PR TITLE
Limit the printed blocks when converting MasterWorkerInfo to string

### PR DESCRIPTION
**Pull request by jiacheliu3**
Addresses https://github.com/Alluxio/alluxio/issues/11729

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java [hunk 1]
```diff
@@ -375,10 +378,19 @@ public boolean hasLostStorage() {
...
+    if (!LOG.isDebugEnabled()) {
+      blocks = (mBlocks.size() < BLOCK_SIZE_LIMIT) ? mBlocks :
+              mBlocks.stream().limit(BLOCK_SIZE_LIMIT).collect(Collectors.toList());
+    }
     return MoreObjects.toStringHelper(this).add("id", mId).add("workerAddress", mWorkerAddress)
-        .add("capacityBytes", mCapacityBytes).add("usedBytes", mUsedBytes)
-        .add("lastUpdatedTimeMs", mLastUpdatedTimeMs).add("blocks", mBlocks)
-        .add("lostStorage", mLostStorage).toString();
+            .add("capacityBytes", mCapacityBytes).add("usedBytes", mUsedBytes)
+            .add("lastUpdatedTimeMs", mLastUpdatedTimeMs).add("blocks", mBlocks)
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java:389]: This adds `blocks` twice?

Also, if the list was actually truncated, can we label it like `blocks-truncated` or something that distinguishes it from the full blocks list?

**Review by gpang — CHANGES_REQUESTED** [review]
@jiacheliu3 Thanks!

File: core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java (hunk 1 above)
Comment (jiacheliu3) [reply core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java]: oops i overlooked that field

Yea makes sense to me `blocks-truncated` would be a better name.

**Review by gpang — APPROVED** [review]
LGTM

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**